packages/strapi-plugin-content-manager/admin/src/translations/en.json

```diff
@@ -85,6 +85,7 @@
   "containers.SettingPage.editSettings.description": "Drag & drop the fields to build the layout",
   "containers.SettingPage.editSettings.entry.title": "Entry title",
   "containers.SettingPage.editSettings.entry.title.description": "Set the displayed field of your entry",
+  "containers.SettingPage.editSettings.relation-field.description": "Set the displayed field in both the edit and list views",
```

Comment:
@soupette quick question to better understand.
Why did you create a new translation and did not update the one we used for this field? Is the original one used in other places?